### PR TITLE
New template (slim) and output improvements

### PR DESCRIPTION
**Pull request by carstene1ns**
- Easier Setup (`make`)
- Cleaner Site skeleton (no unnecessary markup)
- Syntax highlighting:

- Better tables:

### CODE REVIEW

**Comment by carstene1ns** [comment]
jenkins: test this please.

**Comment by carstene1ns** [comment]
jenkins: test this please.

**Comment by Ghabry** [comment]
oh that is nice :).
The only nit I have is that in the code blocks the text after the comment (\#) is hard to read (bad contrast)

**Comment by carstene1ns** [comment]
Yeah, that is the monokai theme. I am open to suggestions (all pygments themes are supported, e.g. https://stylishthemes.github.io/Syntax-Themes/pygments/)

**Comment by Ghabry** [comment]
I have kinda bad color vision but Github Dark looks good to me. Sufficient contrast for me.

**Comment by carstene1ns** [comment]
jenkins: test this please
(somehow my job changes have been lost)

**Comment by Ghabry** [comment]
> somehow my job changes have been lost

Strange. Didn't touch anything.

Lgtm

File: Gemfile [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
+group :nanoc do
+  gem 'guard-nanoc'    # automatic rebuilds
+  gem 'adsf'           # tiny webserver
+  gem 'puma'           # webserver backend
 end
 
 group :test do
-  gem 'nokogiri'
-  gem 'w3c_validators'
+  gem 'nokogiri'       #
```
Comment (Ghabry) [inline Gemfile:18]: nokogiri is the HTML parser in case you want to add a comment here. Love this library, use it for all my quick'n'dirty parsing tasks.

**Pull request merged**